models/course_prerequisite.rb

```diff
@@ -4,5 +4,5 @@
 class CoursePrerequisite < Sequel::Model
   many_to_one :course
   one_to_one :prerequisite
-  one_to_many :coursePrerequisiteReactions
+  one_to_many :coursePrerequisiteReaction
```

Comment:
I suppose that the field name following "one_to_many " tag should be plural so it should have an "s" at the end of the word?